repository/MonticelloFileTree-Core.package/MCFileTreeRepository.class/instance/goferVersionFrom..st

```diff
@@ -1,10 +0,0 @@
-as yet unclassified
-goferVersionFrom: aVersionReference
-    "until we no longer find .tree directories in the wild"
-
-    ((self readableFileNames collect: [ :fileName | self fileDirectoryOn: fileName ])
-        select: [ :packageDirectory | self fileUtils directoryExists: packageDirectory ])
-        collect: [ :packageDirectory | 
-            (self versionInfoForPackageDirectory: packageDirectory) name = aVersionReference name
-                ifTrue: [ ^ self loadVersionFromFileNamed: (self fileUtils directoryName: packageDirectory) ] ].
-    ^ nil
```

Comment:
I think this method is still needed ... it's sent from GoferResolvedReference>>version in GemStone and presumably it is still used on other platforms as well ...